clients/cloud/csharp/README.md

```diff
@@ -0,0 +1,90 @@
+﻿# Overview
+
+Produce messages to and consume messages from [Confluent Cloud](https://www.confluent.io/confluent-cloud/) using the .NET Producer and Consumer.
+
+
+# Prerequisites
+
+* .NET Core 2.1 or higher to run the demo application.

```

Comment:
Please provide link to install instructions (e.g. https://github.com/confluentinc/confluent-kafka-dotnet)